c/build_all/linux/setup.sh

```diff
@@ -12,7 +12,7 @@ repo_name_from_uri()
 }
 
 scriptdir=$(cd "$(dirname "$0")" && pwd)
-deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git"
+deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake unzip openjdk-7-jre"
```

Comment:
Unfortunately I don't remember the exact message, it was something like "JAVA is not defined". I had it with both raspbian and resin builder (which does docker build based on debian AFAIK).